.changeset/smooth-cameras-help.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/octicons': minor
```

Comment:
This probably doesn't need a changeset

Reply:
Yeah probably, it's there to trigger a new release so we can replace the outdated one sooner rather than later.

Reply:
I'll remove it. It should have been a `patch` anyway 🤦